nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/io/marshallers/json/AggregateJsonWriter.java

```diff
@@ -68,6 +72,18 @@
 
     private static final Log log = LogFactory.getLog(AggregateJsonWriter.class);
 
+    /** Fake schema for system properties usable as a page provider aggregate */
+    protected static final Schema SYSTEM_SCHEMA = new SchemaImpl("system", null);
+
+    static {
+        SYSTEM_SCHEMA.addField("ecm:mixinType", new ListTypeImpl("system", "", StringType.INSTANCE), null, 0, null);
```

Comment:
Could you use constants from `NXQL` for all `ecm:*` strings?

Reply:
But this is not NXQL... this is the name we give to system properties in other contexts and are not necessarily the exactly the same as NXQL ones. This kind of namespace for system properties should be established somewhere else in the core.